package.json

```diff
@@ -15,14 +15,14 @@
     "git graph"
   ],
   "scripts": {
-    "build:vite": "ts-node-esm --transpileOnly .vite/build.ts",
-    "build:mermaid": "pnpm build:vite --mermaid",
-    "build:viz": "pnpm build:mermaid --visualize",
+    "build": "pnpm run -r clean && pnpm build:esbuild && pnpm build:types",
+    "build:esbuild": "pnpm run -r clean && ts-node-esm --transpileOnly .esbuild/build.ts",
```

Comment:
Both `build` and `build:esbuild` have `pnpm run -r clean`, which would doable clean. I removed the one in `build` in branch adding `info` diagram.

Reply:
Thanks. Removed :)

Reply:
After more testing, it would better to make the `clean`  in `build:esbuild` (and `dev` if we need).